### PR TITLE
add MetaBase class to model_hub

### PR DESCRIPTION
**Pull request by borft**

### CODE REVIEW

File: analysis/notebooks/model-hub-demo-notebook.ipynb [hunk 1]
```diff
@@ -219,7 +239,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.4"
```
Comment (borft) [inline analysis/notebooks/model-hub-demo-notebook.ipynb:222]: not sure what triggered this, perhaps my local python version is lower?

File: analysis/notebooks/model-hub-demo-notebook.ipynb [hunk 2]
```diff
@@ -80,6 +80,26 @@
     "df.model_hub.aggregate.unique_users().to_pandas()"
    ]
   },
+  {
+   "attachments": {},
```
Comment (borft) [inline analysis/notebooks/model-hub-demo-notebook.ipynb:84]: add cell with explanation and example of `to_metabase()`

File: docker-compose.yaml [hunk 3]
```diff
@@ -59,6 +59,8 @@ services:
     image: postgres:latest
     networks:
       - obj
+    ports:
+      - 5432:5432
```
Comment (thijs-obj) [inline docker-compose.yaml:63]:
```suggestion
      - "127.0.0.1:5432:5432"
```

Comment (jansentom) [reply docker-compose.yaml:63]: Picked up in: https://github.com/objectiv/objectiv-analytics/pull/343

**Pull request merged**